src/context/uuid.rs

```diff
@@ -35,7 +31,9 @@ impl ContextUuidExt for Context {
 	Ok(data)
     }
 
-    fn device_uuids(&self) -> Result<Vec<[u8; 16]>, Self::Error> {
+    /// Returns the UUIDs of the devices being used by this context. Useful to ensure
```

Comment:
Can you put this onto two lines with one empty line between? So that the first sentence is shown in the preview and the rest isn't.